packages/opencensus-core/src/trace/config/types.ts

```diff
@@ -32,6 +33,8 @@ export interface BufferConfig {
 
 /** Defines tracer configuration parameters */
 export interface TracerConfig {
+  /** A set of attributes each in the format [KEY]:[VALUE] */
+  attributes?: Attributes;
```

Comment:
maybe calling it `spanAttributes`? `defaultSpanAttributes`?

Reply:
Another option is either `constantAttributes` or `defaultAttributes`.

Reply:
Thanks for the change, would you mind to update CHANGELOG.md with this new feature?